c/src/point_one/polaris/polaris.c

```diff
@@ -543,26 +554,40 @@ int Polaris_Work(PolarisContext_t* context) {
 #endif
 
   if (bytes_read < 0) {
+    // Did we hit a read timeout? This is normal behavior (e.g., brief loss of
```

Comment:
It may be worth clarifying here that these read timeouts are separate and shorter than the "application level" timeouts (i.e. the 30 seconds in the example)

Reply:
Sure, can do